features/dashboard/task.feature

```diff
@@ -17,8 +17,10 @@ Feature: dashboard test set
 		And <userRole> clicks on Private-task checkbox
 		And <userRole> clicks Add-task-submit button
 		And <userRole> goes to tasks page
-		And <userRole> filter by '<courseName>'
-		When <userRole> sees that task with name '<taskName>' is visible on the list
+		And <userRole> clicks on drafts tab
+		#And <userRole> filter by '<courseName>'
+		Then <userRole> sees '<taskName>' in the list
+		#When <userRole> sees that task with name '<taskName>' is visible on the list
```

Comment:
We should create a ticket to remove the step "#Then <userRole> should see that task with name '<taskName>' is not visible on the list" and also the methods and so on behind it.
If we aren't able to solve the filter issue, it should be added as well.

Reply:
Will be done within BC-1008.